IronMQ.class.php

```diff
@@ -778,20 +780,25 @@ public function getMessagePushStatuses($queue_name, $message_id)
      *
      * Example:
      * <code>
-     * $ironmq->deleteMessagePushStatus("test_queue", $message_id, $subscription_id)
+     * $ironmq->deleteMessagePushStatus("test_queue", $message_id, $reservation_id, $subscriber_name)
      * </code>
      *
      * @param string $queue_name
      * @param string $message_id
-     * @param string $subscription_id
+     * @param string $reservation_id
+     * @param string $subscriber_name
      * @return mixed
      */
-    public function deleteMessagePushStatus($queue_name, $message_id, $subscription_id)
+    public function deleteMessagePushStatus($queue_name, $message_id, $reservation_id, $subscriber_name)
```

Comment:
I assume, that this method must be renamed to something like `deletePushMessage`.